### ISSUE TITLE
Echo Arena - Party

### ISSUE THREAD

**Issue by AVRstation** [LibreVR/Revive#751]
any news about support party?

**Comment by Armada651** [comment]
It's an Oculis Platform incompatibility with Revive, there's not much I can do. The only thing that could be done is hacking full Vive support into the Oculus store, which is a long shot.

**Comment by Werbel** [comment]
Sooooo we are not able to use party system? I wanted to win that ELS matches :<

**Comment by Armada651** [comment]
Not yet, after version 1.2 is out I'll take an in-depth look at the problem to see if there's a hack I can do.

**Comment by Werbel** [comment]
Fair enough. But there is any chance that will it be done before this weekend(9-10.09.2017)?

It would be great to know, I must let know my Echo Arena ESL team if I'll be eligible for league matches.

1.2 beta does not affect this probably?

**Comment by Armada651** [comment]
This is not affected by the 1.2 beta. There is currently one known workaround that involves using an actual Oculus sensor. If you happen to own a DK2 you can use the DK2 camera.

A solution that doesn't require hardware is a little bit more difficult and it's unlikely I'll have that done before the weekend.

**Comment by gsparx** [comment]
FWIW I've been able to join a party with another player also in revive in Echo Arena.  It was a bit spotty, but worked decently for a private match.  Haven't tried a public match with a party, but I imagine it would still work OK.

**Comment by dashnine** [comment]
Tested in public matches this evening.  We had pretty regular disconnects during the party wait, and cancelled repeatedly attempting to get in as a group.  If there's enough space in the match once you start, you can invite friends into it--but that's very hit-or-miss (since you can end up in a full party).

Disconnects happened at random, apparently--an arbitrary person would be removed at any given time.  

For what it's worth, I eagerly await a solution.  As a side note--would just having and plugging in an oculus sensor actually fix the problem?  Just for the person with the camera, or would it affect the whole party if the leader had the camera?

**Comment by noyesa0521** [comment]
Is there any update with this issue? I'm still having issues joining a match with my buddy, he is able to join the arena but then I don't join with him :/
